pow-migration/README.md

```diff
@@ -1,25 +1,42 @@
 # Nimiq Proof-of-Work Migration
 
-This repository contains a set of tools and utilities to help the migration process from Nimiq Proof-of-Work to Nimiq
-Proof-of-Stake (PoS).
-The intended users of these utilities are nodes who want to become validators in the Nimiq PoS chain.
+This repository contains a tool for the migration process from Nimiq Proof-of-Work
+to Nimiq Proof-of-Stake (PoS).
+The intended users of this utility are nodes who want to become validators in the
+Nimiq PoS chain or spectator nodes that want to check and reproduce the migration
+process.
 The functionality in this repository includes:
 
-- Nimiq Proof-of-Stake Genesis builder: This tool is used to create the Nimiq POS Genesis block, which will include the
-  account state (balances) from the current Nimiq POW Chain. It is worth to note that this block would be a continuation
-  of the Nimiq POW chain, with the addition of the first validator set and the Nimiq POW balances, i.e.: this block
-  would constitute the first election block of the Nimiq PoS chain.
-- Validator readiness: Sends a validator ready transaction to signal that the validator is ready to start the migration
-  process and also monitors the PoW chain for other validators readiness signal. When enough validators are ready, then
-  an election block candidate is automatically elected to be used as the final PoW state to be migrated.
-- Nimiq PoS Wrapper: Starts the Nimiq PoS client when the minimum number of block confirmations for the election
-  candidate is satisfied.
-- Nimiq PoS history builder: Migrates the Nimiq PoW history into Nimiq PoS. This is only necessary for history nodes who
-  want to include and support the full history of the Nimiq PoW and PoS chain.
-- Nimiq PoS state builder: Migrates the Nimiq PoW state into Nimiq PoS.
+- Proof-of-Stake Genesis builder: This tool is used to create the Nimiq PoS
+  Genesis block, which will include the account state (balances) from the current
+  Nimiq PoW chain.
+- Validator readiness: Sends a validator ready transaction to signal that the validator
+  is ready to start the migration process and also monitors the PoW chain for other
+  validators readiness signal. When enough validators are ready, the PoS client can
+  start.
+- PoS history builder: Migrates the Nimiq PoW history into Nimiq PoS. This is a requisite
+  for the genesis block since its history root is calculated from this history.
+- Nimiq PoS state builder: Migrates the Nimiq PoW state into Nimiq PoS. This is a
+  requisite for the genesis block since its accounts tree root is calculated from
+  this state.
+- Migration binary: Starts the Nimiq PoS client when enough validators signaled a
```

Comment:
Starts the Nimiq PoS client when enough validators are ready in the expected block window